pbfs/Dockerfile

```diff
@@ -0,0 +1 @@
+RUN mkdir /tmp/pbfs
```

Comment:
Hum that cannot work no ?

My previous point was to suggest to put this in the k8s files (and aren't the volumes autocreated ?)